lib/kube/proxy/forwarder.go

```diff
@@ -1898,12 +1901,14 @@ func (f *Forwarder) getClientCreds(ctx authContext) *tls.Config {
 	if !validClientCreds(f.cfg.Clock, c) {
 		return nil
 	}
+	f.log.Debugf("Returning cached client credentials with key: %v", ctx.key())
```

Comment:
This was useful for debugging but I think this might be spammy IRL, I'd probably remove these debug logs here and below now.